### PR TITLE
Updated test cases and complete cov_mx

### PR DESCRIPTION
**Pull request by K3ra-y**

### CODE REVIEW

File: R/cov_mx.R [hunk 1]
```diff
@@ -1,12 +1,66 @@
...
+  }
+
+  # return NA if The length of input equal to 1
+  else if (length(x) == 1L){
+    return (NA)
+  }
+
+  # return error if matrix is not 2D
+  if (dim(x)[2] == 1){
+    stop("'x' must be 2D matrix")
```
Comment (xiaoweideng) [inline R/cov_mx.R]: If input dimension is not 2, then the function should `return (NA)`.

File: R/cov_mx.R [hunk 2]
```diff
@@ -1,12 +1,66 @@
...
+  }
+  else if (dim(x)[2] > 2){
+    stop("'x' must be 2D matrix")
```
Comment (xiaoweideng) [inline R/cov_mx.R]: same here

File: R/cov_mx.R (hunk 1 above)
Comment (xiaoweideng) [reply R/cov_mx.R]: Also, change the corresponding test cases to `expect_true(is.na(cov_mx(single_row_x)))`

**Comment by mru4913** [comment]
All the test passed, proved and merged the pull request

**Comment by K3ra-y** [comment]
@mru4913 and @xiaoweideng  Please see the changes

**Pull request merged**